frontend/src/components/dashboard/EmailForwardingModal.tsx

```diff
@@ -108,14 +101,13 @@ const ConfirmModal = (props: Props) => {
           <input
             value={inputValue}
             onChange={(e) => setInputValue(e.target.value)}
-            onBlur={onBlur}
             aria-label={l10n.getString("profile-label-edit-2")}
             ref={inputRef}
             className={styles["label-input"]}
             placeholder={l10n.getString(
               "profile-free-onboarding--copy-mask-placeholder-relay-email-mask",
             )}
-            type="text"
+            type="email"
```

Comment:
I'm not sure about this one. It seems like an email input to me, but that may have undesired behavior.

Reply:
It will require them to input an email instead of any text - the built-in validation of an html input field.

Reply:
ONLY thing I can maybe foresee is the keyboard some devices popup might be optimized for email input, but I think that's a good thing here. Users are expected to copy/paste the email anyway. Oh, and autofill.